ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1198,6 +1204,7 @@ def pingFromServers(self):
     def pingFromUpperTier(self):
         for entry in self.from_t1:
             testutils.send_packet(self, *entry)
+            time.sleep(.001)
```

Comment:
Is it possible to avoid this pause?
Why do we need it?

Reply:
As I mentioned in comment above - from my investigation, it seems that packets sent in a loop (with no delay btw) will be stacked inside one of the system buffers due to huge virtualization (python-vm) and then are pushed out to the network as a big burst having extremely short interpacket gap on the phy level. That's why `swss` really can be not fast enough for FDB and ARP entries insertion (mentioned in the testcase code with [FIXME](https://github.com/Azure/sonic-mgmt/blob/44a781a9374fa8ecac4183b4a105f5a5bf473307/ansible/roles/test/files/ptftests/advanced-reboot.py#L153) clause). Inserting 1ms delay we just re-schedule processes queue and push packets to the network one by one.

The problem appeared to be present even on platforms having rich CPU ( >= 4cores) and from my understanding has a bit more relation to the swss performance. Actually, that's why I pointed that have tested even using ReloadTest::max_nr_vl_pkts == 1000

Reply:
We run ptf inside of the docker container. The docker container works on the host machine. ptf uses interfaces which were inserted there from the host system.
So I don't see virtualization here.
I'd say we're seeing this issue because of some ptf issues.

Let's  remove this change from here as irrelevant to the title of this PR.

Reply:
removed